### ISSUE TITLE
Your package includes ZIPFoundation version 0.9.18 but should pickup 0.9.19 which includes privacy manifests fix

### ISSUE THREAD

**Issue by TanvirShirazi** [NordicSemiconductor/IOS-DFU-Library#535]
Your *NordicDFU* package includes *ZIPFoundation* version 0.9.18 but should pickup 0.9.19 which includes privacy manifests fix otherwise users of _NordicDFU_ will get privacy manifest issue from Apple 

Current:
```swift
 .package(url: "https://github.com/weichsel/ZIPFoundation", exact: "0.9.18")
```
Should be updated to 0.9.19 which includes privacy manifests fix.

**Comment by philips77** [comment]
Fixed and released as 4.15.3. Thank you for creating the issue!

**Comment by Got2Go9** [comment]
[](url)
